Readme.md

```diff
@@ -0,0 +1,48 @@
+# Okta API Access Management Sample Resource Server (ASP.NET Core)
+
+The code sample available in this repository demonstrate the use of [Okta API Access Management](http://developer.okta.com/docs/api/resources/oauth2.html) for authorizing access to a backend API using Okta's Authorization Servers.
+
+This ASP.NET Core OAuth 2.0 Resource Server will only accept calls to its TodoController if 
+1. the incoming requests contains the proper _Bearer_ OAuth token (i.e. properly validated as originating from the expected Okta organization, OpenID Connect client application and audience)
+2. the OAuth token includes a "todolist.read" scope generated by an Okta Authorization Server for the user performing the API access request to the TodoController.
+
+## Development environment instructions
+
+This code samples was written with Visual Studio 2015 Community Edition Update 3 and we strongly suggest that you use the same development environment (or any other paid-for edition of Visual Studio 2015).
```

Comment:
Also, don't know about the "strongly suggest" part - maybe just let that be implied and just do a hard stop after the name of the software.